inputs/autocomplete-item.vue

```diff
@@ -29,18 +39,30 @@
     v-bind:class="{ 'selected': isActive }"
     type="button"
     @click.stop.prevent="select(value)">
-    {{ value }}
+    <span class="item-value">{{value}}</span>
+    <span class="item-actions">
+      <ui-icon-button v-show="canDestroy" icon="clear" :tooltip="`Remove ${value} from list`" color="default" size="small" type="secondary" @click.stop="destroy(value)"></ui-icon-button>
```

Comment:
You're using _destroy_ and _remove_ interchangeably. It would reduce developer mental overhead if a single term (say, _remove_) was used everywhere.